duckduckgo/iOS
# improve keyboard shortcuts

Task/Issue URL: https://app.asana.com/0/392891325557410/1183224745168980
Tech Design URL:
CC:

**Description**:

Add new keyboard shortcuts:

* ⌘G / ⌘⇧G - Find next / previous
* ⌘R - Reload
* ⌘ Click / ⌘⇧ Click - Open in new background/ foreground tab (requires iOS 13.4)
* left/right arrows for navigating tab switcher (not showing in shortcuts help)
* ⌘ ⌥ Tab - Open tab switcher
* ⌘ P - Print
* ⌘ D - Add Bookmark
* ⌘ shift D - Add Favorite 

Fixes a bug when using keyboard shortcuts to navigate tabs.

**Steps to test this PR**:
1. Press and then press and hold down ⌘ to show the shortcuts screen.  Make sure it looks complete.
1. Using an iPad + keyboard  (could be simulator with "I/O > Input > Send Keyboard Input to Device" enabled, try out the above commands.
1. Ensure previous commands still work
1. Open tab switcher and confirm help screen looks correct and commands still work as expected
1. Open a few tabs, leave a couple on the home screen, and browse on the others, ensure that next / previous tab work as expected.



**Device Testing**:


**OS Testing**:



---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
